vcd/resource_vcd_vapp_emtpy_vm_test.go

```diff
@@ -352,11 +352,13 @@ resource "vcd_vapp_vm" "{{.VMName}}" {
     ip_allocation_mode = "NONE"
     ip                 = ""
     name               = ""
+    connect            = false
```

Comment:
The test itself doesn't check field values

Reply:
added